### PR TITLE
docs(getting_started): Improve oh-my-zsh completion instructions

### PR DESCRIPTION
**Pull request by jajaperson**
It is possible for a user to define a custom directory for custom `oh-my-zsh` plugins using the environment variable `$ZSH_CUSTOM`. This pull request updates [this](https://deno.land/manual@v1.6.3/getting_started/setup_your_environment#shell-autocomplete) part of the docs to account for this.

### CODE REVIEW

**Comment by ry** [comment]
is ZSH_CUSTOM always defined?

**Comment by jajaperson** [comment]
Yep, oh-my-zsh automatically sets it if it isn't already ([see here](https://github.com/ohmyzsh/ohmyzsh/blob/86f805280f6a8cf65d8d0a9380489aae4b72f767/oh-my-zsh.sh#L23))

**Comment by lucacasonato** [comment]
@jajaperson can you rebase on master?

File: docs/getting_started/setup_your_environment.md [hunk 1]
```diff
@@ -58,8 +58,8 @@ and then `compinit` to generate them again.
 Example (zsh + oh-my-zsh) [recommended for zsh users] :
 
 ```shell
-mkdir ~/.oh-my-zsh/custom/plugins/deno
-deno completions zsh > ~/.oh-my-zsh/custom/plugins/deno/_deno
+mkdir $ZSH_CUSTOM/deno
```
Comment (kt3k) [inline docs/getting_started/setup_your_environment.md:61]: This might be `$ZSH_CUSTOM/plugins/deno`?

Comment (jajaperson) [reply docs/getting_started/setup_your_environment.md:61]: Oh yeah you are correct

**Comment by jajaperson** [comment]
whoops. i broke it

**Pull request closed without merging**